src/API/sim_utils.jl

```diff
@@ -1,13 +1,17 @@
-function load_netlist(netlist::Array{String})
+function load_netlist(netlist::AbstractArray{T}) where T <: AbstractString
     append!(netlist, ["C_NULL"])
     t = netlist |> ngSpice_Circ
 end
 
-function load_netlist(netpath::String)
+function load_netlist(netpath::AbstractString)
     netlist = readlines(netpath)
     load_netlist(netlist)
 end
 
+function load_netlist_str(netlist::AbstractString)
```

Comment:
The segfault is introduced as the normal `split()` splits on `isspace`. Hence every word becomes a circuit line which Ngspice doesn't understand as one. Also, this keeps the `\n` in the circuit. 

In my commit, I'll suitably modify.